string CIRCULAR="circular"
string RECTANGULAR="rectangular"

# Goal
string command
string table_type
---
# Result
bool success
string error_msg
---
# Feedback
bool status
